### DISCUSSION TITLE
Question: Use of skip auth

### DISCUSSION THREAD

**Discussion by N8-I** [IvanJosipovic/OIDC-Guard#94] [Q&A] [answered]
I was wondering what the use case for using the skip auth query parameters on the `auth` endpoint. At service level it looks scary 👻. As I understand it, this would have to be set by the ingress controller when it makes a call to the `auth` endpoint.  I have not been able to find any documentation on how the ingress controller would make use of this parameter. I was hoping you could shed some light on it for me.

**Comment by IvanJosipovic** [comment, marked as answer]
Hey, thanks for the question, I definitely need to do more work on the docs :D

"At service level it looks scary 👻."

I think some of you confusion is around the overall flow. Each ingress can customize what if any endpoints skip auth or what claims are expected. This is useful for many reasons, for example, making a part of the application anonymous while protecting the rest of the app.

https://github.com/IvanJosipovic/OIDC-Guard/wiki/Endpoints#skip-auth
